File: conditionals/time.py
#johann Hines, How to get the time of day. 

import time
#first instance of time in programming
#print(time.gmtime(0))

# #current time in se
current =time.time()

# current time as we are used to seeing it 
now = time.ctime(current)
print(now)

#get just the hour
local_time = time.localtime(current)
minutes = local_time.tm_min
Hour = local_time.tm_hour
print(local_time)
print (Hour)

File: conditionals/time.c
 //johann hines, template
#include <stdio.h>
#include <time.h>


int main(void){
    // time since jan 1 1970
    time_t seconds; 

    seconds = time(NULL);
    //printf("seconds since january 1, 1970 = %d\n", seconds);
    

// current time
    time_t rawtime; 
    struct tm * timeinfo;

    time(&rawtime);
    timeinfo = localtime(&rawtime);
    printf("Our current time and date is %s\n", asctime(timeinfo));


// current hour
time_t now = time(NULL);
struct tm *tm_struct = localtime(&now);
int hour = tm_struct->tm_hour;
printf("%d\n", hour);
return 0;
}